TypeRaceAir/race.py

```diff
@@ -0,0 +1,63 @@
+import typing
+
+
+class Race():
+
+    def __init__(self, sentece: str, players: typing.List[Player]):
+        self.sentece = sentece
+        self.players = players
+
+
+class Player():
+    pass
+
+
+class PlayerRaceResult():
+    def __init__(self, race: Race, player: Player, submit_timestamp: float, submit_sentence: str):
+        self.race = race
+        self.submit_sentence = submit_sentence
+        self.player = player
+        self.submit_timestamp = submit_timestamp
+
+    @property
+    def words_mistake(self,) -> int:
+        '''The number of words that the user typed incorrectly. '''
+        return len(self.race.sentece.split()) - self.words_correct
+
+    @property
+    def words_correct(self,) -> int:
+        '''The number of words that the user typed correctly. '''
+        submitted_words = self.submit_sentence.split()
+        target_words = self.race.sentece.split()
+        return sum(
+            1
+            for cur_submitted, cur_target in zip(submitted_words, target_words)
+            if cur_submitted == cur_target
+        )
+
+    @ property
```

Comment:
Please lint your files before check-in
```suggestion
    @property
```